--- nget-0.18.2/Makefile.in.orig	Sat May 18 00:33:56 2002
+++ nget-0.18.2/Makefile.in	Wed May 22 20:02:49 2002
@@ -13,12 +13,12 @@
 mandir=@mandir@
 install=@INSTALL@
 install_data=${install} -m 0644
-install_bin=${install} -s -m 0755
+install_bin=${install} -m 0755
 install_dir=${install} -d
 
 all: .autostuff nget ngetlite
 
-INSTALLDIRS=$(bindir) $(mandir)/man1
+INSTALLDIRS=$(DESTDIR)$(bindir) $(DESTDIR)$(mandir)/man1
 
 OBJS=nget.o prot_nntp.o sockpool.o sockstuff.o cache.o dupe_file.o file.o termstuff.o misc.o strtoker.o strreps.o myregex.o datfile.o etree.o nrange.o log.o server.o @UULIB@
 nget: $(OBJS) .depend 
@@ -48,12 +48,12 @@
 	$(install_dir) $@
 
 install_nget: $(INSTALLDIRS) nget
-	$(install_bin) nget $(bindir)
-	$(install_data) nget.1 $(mandir)/man1
+	$(install_bin) nget $(DESTDIR)$(bindir)
+	$(install_data) nget.1 $(DESTDIR)$(mandir)/man1
 
 install_ngetlite: $(INSTALLDIRS) ngetlite
-	$(install_bin) ngetlite $(bindir)
-	$(install_data) ngetlite.1 $(mandir)/man1
+	$(install_bin) ngetlite $(DESTDIR)$(bindir)
+	$(install_data) ngetlite.1 $(DESTDIR)$(mandir)/man1
 
 install: install_nget install_ngetlite
 
