diff -ur nget-0.14.orig/Makefile.in nget-0.14/Makefile.in
--- nget-0.14.orig/Makefile.in	Fri Aug  3 03:36:40 2001
+++ nget-0.14/Makefile.in	Mon Aug  6 22:52:07 2001
@@ -39,11 +39,11 @@
 	$(CXX) $(CXXFLAGS) -MM *.cc > .depend
 
 install_nget: nget
-	$(install) -s -o root -g bin -m 0755 nget $(bindir)
-	$(install) -o root -g bin -m 0644 nget.1 $(mandir)/man1
+	$(install) nget $(DESTDIR)$(bindir)
+	$(install) nget.1 $(DESTDIR)$(mandir)/man1
 
 install_ngetlite: ngetlite
-	$(install) -s -o root -g bin -m 0755 ngetlite $(bindir)
+	$(install) ngetlite $(DESTDIR)$(bindir)
 
 install: install_nget install_ngetlite
 
