--- nget-0.27.1/Makefile.in.orig	2004-12-21 22:45:04.000000000 +0100
+++ nget-0.27.1/Makefile.in	2004-12-21 22:47:55.000000000 +0100
@@ -19,7 +19,7 @@
 mandir=@mandir@
 install=@INSTALL@
 install_data=${install} -m 0644
-install_bin=${install} -s -m 0755
+install_bin=${install} -m 0755
 install_dir=${install} -d
 
 NGET=nget$(EXEEXT)
@@ -27,7 +27,7 @@
 
 all: .autostuff $(NGET) $(NGETLITE)
 
-INSTALLDIRS=$(bindir) $(mandir)/man1
+INSTALLDIRS=$(DESTDIR)$(bindir) $(DESTDIR)$(mandir)/man1
 W32DISTTXTFILES=COPYING Changelog FAQ README.win32 nget.txt ngetlite.txt _ngetrc
 W32DISTBINFILES=$(NGET) $(NGETLITE)
 W32DISTFILES=$(W32DISTTXTFILES) $(W32DISTBINFILES)
@@ -56,12 +56,12 @@
 	$(install_dir) $@
 
 install_nget: $(INSTALLDIRS) $(NGET)
-	$(install_bin) $(NGET) $(bindir)
-	$(install_data) nget.1 $(mandir)/man1
+	$(install_bin) $(NGET) $(DESTDIR)$(bindir)
+	$(install_data) nget.1 $(DESTDIR)$(mandir)/man1
 
 install_ngetlite: $(INSTALLDIRS) $(NGETLITE)
-	$(install_bin) $(NGETLITE) $(bindir)
-	$(install_data) ngetlite.1 $(mandir)/man1
+	$(install_bin) $(NGETLITE) $(DESTDIR)$(bindir)
+	$(install_data) ngetlite.1 $(DESTDIR)$(mandir)/man1
 
 install: install_nget install_ngetlite
 
